packages/core/src/Stores/client-store.js

```diff
@@ -2581,13 +2581,11 @@ export default class ClientStore extends BaseStore {
     }
 
     getChangeableFields() {
-        const get_settings =
-            Object.keys(this.account_settings).length === 0
```

Comment:
There seems to be bit of a logic difference. In existing code, a check was made to see if `this.account_settings` and fetch the value. The error was because for some reason, the `immutable_fields` array was undefined/null. Do u also need to make a call to getSettings if account_settings is not available?

Reply:
this call is made during init() function.
here previously this request didn’t work, because the data wasn’t extracted here. we always took data from this.account_settings